docs/ops.html

```diff
@@ -586,7 +582,7 @@
   socket.request.max.bytes=104857600
   socket.receive.buffer.bytes=1048576
   socket.send.buffer.bytes=1048576
-  queued.max.requests=16
+  queued.max.requests=500
```

Comment:
@junrao I have seen increasing num.replica.fetchers help inter broker replication throughput on medium to large nodes in production clusters. Have you not seen similar?